control/src/lib.rs

```diff
@@ -378,25 +370,24 @@ pub mod pallet {
 			membership_fee: Option<T::Balance>,
 		) -> DispatchResult {
 			let mut org = Orgs::<T>::get(&org_id).ok_or(Error::<T>::OrganizationUnknown)?;
-			// SBP-M2 review: Get rid of Sudo calls
 			// Create entity like a council
 			Self::ensure_root_or_prime(origin, org.prime.clone(), org.org_type.clone())?;
 
 			let args = [prime_id.is_some(), fee_model.is_some(), membership_fee.is_some(),
 						access_model.is_some(), member_limit.is_some(), org_type.is_some()];
 			ensure!(args.iter().any(|x| *x == true), Error::<T>::NoChangesProvided);
 
-			// SBP-M2 review: I do not like these instructions with pattern matching
-			// I would use quick check `is_some`: https://doc.rust-lang.org/std/option/enum.Option.html#method.is_some
-			if let Some(access_model) = access_model.clone() { org.access_model = access_model; };
-			if let Some(org_type) = org_type.clone() { org.org_type = org_type; };
-			if let Some(member_limit) = member_limit.clone() { org.member_limit = member_limit; };
-			if let Some(_) = membership_fee.clone() { org.membership_fee = membership_fee; };
-			if let Some(prime_id) = prime_id.clone() {
+			if access_model.is_some() { org.access_model = access_model.clone().unwrap(); };
```

Comment:
Why this syntax is better than [if let Some](https://doc.rust-lang.org/book/ch06-03-if-let.html), which is common and is a recommendation from the Rust book.